webserver.py

```diff
@@ -15,11 +15,12 @@
 
     @web.lm_unit.lm_unit_webserver.listener('before_server_start')
     def cnf(sanic, loop):
-        logger.debug('before_start {} {} {}'.format(
-                                                    sanic,
-                                                    hex(id(loop)),
-                                                    threading.current_thread().name
-        ))
+        pass
+#        logger.debug('before_start {} {} {}'.format(
+#                                                    sanic,
+#                                                    hex(id(loop)),
+#                                                    threading.current_thread().name
+#        ))
```

Comment:
I would prefer to not keep unused code any longer.

Reply:
removed